### ISSUE TITLE
Kops rolling doesn't wait until masters are terminated / etcd volumes are available again

### ISSUE THREAD

**Issue by johanneswuerbach** [kubernetes/kops#4128]
1. What `kops` version are you running? The command `kops version`, will display
 this information.

```
$ kops version
Version 1.8.0
```

2. What Kubernetes version are you running? `kubectl version` will print the
 version if a cluster is running or provide the Kubernetes version specified as
 a `kops` flag.

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.0", GitCommit:"925c127ec6b946659ad0fd596fa959be43f0cc05", GitTreeState:"clean", BuildDate:"2017-12-16T03:16:50Z", GoVersion:"go1.9.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.11", GitCommit:"b13f2fd682d56eab7a6a2b5a1cab1a3d2c8bdd55", GitTreeState:"clean", BuildDate:"2017-11-25T17:51:39Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
```

3. What cloud provider are you using?

`AWS: us-east-1`

4. What commands did you run?  What is the simplest way to reproduce this issue?

`kops rolling-update cluster NAME --yes`

5. What happened after the commands executed?

One of the to be rolled masters didn't terminate for a long time (~1h), causing the etcd volumes to be still mounted to the terminating machine and not mountable on the replacement instance.

![image](https://user-images.githubusercontent.com/864578/34252719-0af780f4-e645-11e7-94e4-1fb7adadc651.png)

6. What did you expect to happen?

kops failing or waiting until the old master is terminated before trying to start a new one

**Comment by chrislovecnm** [comment]
Yes that would be a good check!

**Comment by justinsb** [comment]
/remove-lifecycle rotten


This may be partially addressed by #5446
